setup.py

```diff
@@ -26,40 +26,38 @@ def load_requirements(path: Path):
 
 requirements = load_requirements(Path(__file__).parent / "requirements.txt")
 
-setup_requirements = [
-    "pytest-runner",
-]
-
-test_requirements = [
-    "pytest>=3",
-]
-
 setup(
     author="DrivenData",
     author_email="info@drivendata.org",
-    python_requires=">=3.5",
+    python_requires=">=3.6",
     classifiers=[
-        "Development Status :: 2 - Pre-Alpha",
+        "Framework :: Jupyter",
         "Intended Audience :: Developers",
-        "Natural Language :: English",
+        "Intended Audience :: Science/Research",
+        "License :: OSI Approved :: MIT License",
+        "Operating System :: OS Independent",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
     ],
-    description="Automatically export Jupyter notebooks to various file formats (.py, .html, and more) on save.",
+    description=(
+        "Automatically export Jupyter notebooks to various file formats (.py, .html, and more) on "
+        "save."
+    ),
     entry_points={"console_scripts": ["nbautoexport=nbautoexport.nbautoexport:app"]},
     install_requires=requirements,
     long_description=readme + "\n\n" + history,
     long_description_content_type="text/markdown",
     include_package_data=True,
-    keywords="nbautoexport",
+    keywords=["nbautoexport", "jupyter", "nbconvert"],
     name="nbautoexport",
     packages=find_packages(include=["nbautoexport", "nbautoexport.*"]),
-    setup_requires=setup_requirements,
-    test_suite="tests",
-    tests_require=test_requirements,
```

Comment:
Just for my own learning, why don't we need lines 60-62?

Reply:
https://setuptools.readthedocs.io/en/latest/setuptools.html This page from the `setuptools` docs is where this stuff is described.

`setup_requires` has a red warning box that says it's discouraged in favor of the PEP-517/PEP-518 new system using `pyproject.toml` that we ran into with `sample-weights`. 

`test_suite` and `tests_require` are both for if we run our tests with `python setup.py tests`. We don't run our tests that way anyways, and `New in 41.5.0: Deprecated the test command.` is everywhere in the `setuptools` docs that has anything to do with running tests.

Reply:
TIL :bow: